plugins/huggingface/api_token.go

```diff
@@ -0,0 +1,88 @@
+package huggingface
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIToken() schema.CredentialType {
```

Comment:
I don't have strong opinions here, but I think it'd be ideal to match the credential name that Hugging Face uses: [User Access Tokens](https://huggingface.co/settings/tokens), or just Access Tokens. If we decide to update the credential name, let's be sure to update the credential name and credential function as well.

Reply:
What do you think, @hculea?

Reply:
Definitely, thanks for flagging this! 😄 Not a bug or anything, but good practice.